app/controllers/figures_controller.rb

```diff
@@ -1,13 +1,21 @@
+# frozen_string_literal: true
```

Comment:
^ why are there a ton of #frozen_string_literal: true on top of a lot of your files?